packages/diffhtml-components/lib/component.js

```diff
@@ -1,7 +1,7 @@
+import process from './util/process';
```

Comment:
`./util/process` has `import { process } from 'diffhtml-shared-internals/lib/process';`

The util simply makes sure that `process` is a global.